glean-core/src/metrics/uuid.rs

```diff
@@ -148,12 +148,11 @@ impl UuidMetric {
     /// # Returns
```

Comment:
All these metrics helpfully include doc comments about `ping_name` which should probably be removed.